domains/certificates/Query.API/API/MeasurementsSyncer/SlidingWindowService.cs

```diff
@@ -174,15 +193,37 @@ private static List<MeasurementInterval> FindMissingIntervals(MeteringPointTimeS
             {
                 if (IsCurrentMeasurementIndexInsideMissingInterval(currentMissingIntervalStart))
                 {
-                    AddMissingInterval(currentMissingIntervalStart!, UnixTimestamp.Create(currentMeasurement.DateFrom), missingIntervals);
+                    AddMissingIntervalUpToThreshold(currentMissingIntervalStart!, UnixTimestamp.Create(currentMeasurement.DateFrom), missingIntervals, minimumAgeThreshold);
                     currentMissingIntervalStart = null;
                 }
             }
         }
 
+        var remainingMissingIntervals = window.MissingMeasurements.Intervals
```

Comment:
If you have missing intervals after the synchronization point, then you must implicitely also have published measurements, how do you know what measurements have already been published after the synchronization point?

Reply:
The logic contained here is for separating missing intervals within the threshold from those after the threshold.

Once a missing interval crosses the threshold it will be visible for the worker.